flzoo/default_config.py

```diff
@@ -7,6 +7,12 @@
         dataset='cifar10',
         # Root path for dataset.
         data_path='./data',
+
```

Comment:
1) Do not modify the default config
2) Do not include the absolute path of your computer. For example, you can put the data to `./data/domain_net`